hassio-google-drive-backup/backup/static/js/settings.js

```diff
@@ -170,6 +170,7 @@ function handleSettingsDialog(data) {
                     </label>
                   </li>`;
     template = template
+      .replace("{id}", addon.slug)
```

Comment:
The view have two `{id}` to replace, and the `string.replace` javascript function only replace one by default; so I call it twice to fix it.